tests/test_module_import.py

```diff
@@ -17,7 +17,9 @@
 
 class TestModuleImport:
     def test_import(self):
-        """ """
+        """
+        RUCIO: Test if Rucio module can be imported without errors.
```

Comment:
I personally would add `D419` (https://docs.astral.sh/ruff/rules/empty-docstring/) to the list of `'tests/*.py'` ignores in `[tool.ruff.lint.per-file-ignores]` in the cfg. I think a lot of tests are self-documented through their function name, and enforcing a docstring for them might be a bit too much (but it can be welcome in case it's useful; I think it's useful in the case of `test_import` here, for example)